### PR TITLE
Update osgeo import syntax, bugfix

### PR DESCRIPTION
**Pull request by ehusby**
Includes an important bugfix for the issue brought up by a kind user: https://github.com/PolarGeospatialCenter/pgcdemtools/issues/12

### CODE REVIEW

File: apply_setsm_registration.py [hunk 1]
```diff
@@ -198,8 +198,9 @@ def apply_reg(srcfp, args):
         if line.startswith('Translation Vector (dz,dx,dy)(m)='):
             trans_vector = line.strip().split('=')[1]
             trans_vector = trans_vector.split(', ')
-            dx, dy, dz = [float(parm.strip()) for parm in trans_vector]
-            logger.info("Translation Vector (dx, dy, dz) (m) = {}, {}, {}".format(dx, dy, dz))
+            dz, dx, dy = [float(parm.strip()) for parm in trans_vector]
```
Comment (clairecporter) [inline apply_setsm_registration.py:201]: Why this change?  How does it impact the translation vector appliying code?

Comment (ehusby) [reply apply_setsm_registration.py:201]: The change from
`dx, dy, dz = [float(parm.strip()) for parm in trans_vector]`
to
`dz, dx, dy = [float(parm.strip()) for parm in trans_vector]`
is the critical bugfix, as mentioned by the user in https://github.com/PolarGeospatialCenter/pgcdemtools/issues/12.

**Review by clairecporter — APPROVED** [review]
